src/Modules/Settings/Dnn.PersonaBar.Extensions/Extensions.Web/src/components/InstallExtensionModal/index.jsx

```diff
@@ -128,6 +128,7 @@ class InstallExtensionModal extends Component {
             selectedLegacyType: null,
             alreadyInstalled: false
         });
+        props.onCancel();
```

Comment:
@mgatmaytan, can you please help me on this review. The cancelInstall receive a cancelRevertStep but it look like this is always set to false + for some reason the callback in clearParsedInstallationPackage is not called or does not work. Can you please help on review the @pberna fix and cleanup as part of this PR if those code is not needed. Thks.